site/content/docs/cli/using-constructs.md

```diff
@@ -115,6 +115,58 @@ All resources you can create and manage using the Checkly CLI are derived from "
 You can use standard JS/TS programming to use these constructs to create the monitoring setup of your
 choice. Loops, variables, if-statements, file imports, extensions etc.
 
+### Using ECMAScript modules files
+
+If your project uses ECMAScript modules files, you can specify `type: "module"` in your `package.json` file or use `.mjs` file extensions. ECMAScript modules files can be used to create resources or as check's script dependencies.
+
+> Note that `__dirname` and `__filename` don't exist in ECMAScript so you have to adapt your setup/teardown script references using relative paths.
+> Also, as the setup/teardown script dependencies are run under a sandboxed environment, you must use CommonJS (or TypeScript project) for all the second-level dependencies for API and Browser checks.
+
+#### Example using top-level `await` supported in ECMAScript
+
+```js
+// __checks__/api.check.js
```

Comment:
should this not be `api.check.mjs`?

Reply:
As I was talking about ECMAScript projects (`type: "module"` configure in your `package.json`) it's the same. I'll change the extensions just in case users want to experiment with a `commonjs` project using just `.mjs` files.